Source/Core/DiscIO/CISOBlob.cpp

```diff
@@ -5,20 +5,20 @@
 #include <algorithm>
 #include <cstdio>
 #include <memory>
+#include <utility>
 
 #include "Common/CommonTypes.h"
 #include "Common/FileUtil.h"
 #include "DiscIO/CISOBlob.h"
 
 namespace DiscIO
 {
-static const char CISO_MAGIC[] = "CISO";
-
-CISOFileReader::CISOFileReader(std::FILE* file) : m_file(file)
+CISOFileReader::CISOFileReader(File::IOFile&& file) : m_file(std::move(file))
```

Comment:
Just recalled this, but IOFile is already a move-only type by default, so you can just take this by value, similar to how unique_ptrs taking ownership in function signatures are taken by value.